src/cabanamd_impl.h

```diff
@@ -66,7 +66,11 @@ void CbnMD<t_System, t_Neighbor>::init( InputCL commandline )
     if ( system->do_print )
     {
         using exe_space = typename t_System::execution_space;
+#ifndef KOKKOS_ENABLE_HIP
         exe_space::print_configuration( std::cout );
+#else
+        std::cout << "Using Kokkos::HIP" << std::endl;
+#endif
```

Comment:
why do you not rely on `Kokkos::Experimental::HIP::print_configuration()`?

Reply:
Missed that - thanks

Reply:
I thought I had left in a debug print, but I had added this because Kokkos aborts if the `HIP::print_configuration` is called

Reply:
Oh yeah... https://github.com/kokkos/kokkos/blob/7a1265fd58b503bcf1a0f4a316c50b44fcf089c6/core/src/HIP/Kokkos_HIP_Instance.cpp#L106
:(

Reply:
FYI https://github.com/kokkos/kokkos/pull/3080